cmd/livepeer/livepeer.go

```diff
@@ -133,7 +133,7 @@ func parseLivepeerConfig() starter.LivepeerConfig {
 	cfg.SelectPriceExpFactor = flag.Float64("selectPriceExpFactor", *cfg.SelectPriceExpFactor, "Expresses how significant a small change of price is for the selection algorithm; default 100")
 	cfg.OrchPerfStatsURL = flag.String("orchPerfStatsUrl", *cfg.OrchPerfStatsURL, "URL of Orchestrator Performance Stream Tester")
 	cfg.Region = flag.String("region", *cfg.Region, "Region in which a broadcaster is deployed; used to select the region while using the orchestrator's performance stats")
-	cfg.MaxPricePerUnit = flag.Int("maxPricePerUnit", *cfg.MaxPricePerUnit, "The maximum transcoding price (in wei) per 'pixelsPerUnit' a broadcaster is willing to accept. If not set explicitly, broadcaster is willing to accept ANY price")
+	cfg.MaxPricePerUnit = flag.String("maxPricePerUnit", *cfg.MaxPricePerUnit, "The maximum transcoding price per 'pixelsPerUnit' a broadcaster is willing to accept. If not set explicitly, broadcaster is willing to accept ANY price. Can be specified in wei or a custom currency in the format <price><currency> (e.g. 0.50USD). When using a custom currency, a corresponding price feed must be configured with -priceFeedAddr")
```

Comment:
I see a couple of options:
- Allow using the `e` notation for the price, so that it could be at least `-maxPricePerUnit=7e-13USD` 
  - Notice that we need to adapt the parsing function regex and the dynamic update APIs to support this
- Make use of the `pixelsPerUnit` configuration, which acts basically as a denominator for the price here. I'd say we should align on some value that will keep the `maxPricePerUnit` on a reasonable range.
  - For example, if we set the `-pixelsPerUnit` value to `1000000000000` (or `1e12`), we could set the `-maxPricePerUnit=0.70USD` here which feels more natural.

WDYT?

Reply:
@leszko @victorges great callout, I ran into this exact problem when working with Evan yesterday. My personal reaction is that the pixelsPerUnit configuration is a nicer way of handling this, BUT it will need to be well documented

Reply:
Specifically we'd need to recommend to Os how they should set it. 

Some Os may even keep setting prices in wei (which should still work?)

Reply:
Yeah there's ample support for the "pixels per unit" config, it already existed before this change. It will also be possible to keep setting prices in wei :)

Reply:
Ok, fine with me 👍

Reply:
OK, I've added this commit https://github.com/livepeer/go-livepeer/pull/2981/commits/ed7dc9b7285e4f82ba795636c7dbc97aaaa405e7 which makes sure we can always specify the `pixelsPerUnit` with an exponential notation (livepeer args, livepeer_cli or json file). I think this is enough, with the idea being to set a nice number for `pixelsPerUnit` with an easy syntax like `1e12`, and then the `pricePerPixel` can be a small enough number that should be fine to specify on the regular syntax (we won't support `e` notation for that in some flows (args)).

One other thing I could add would be forcing a different default value for the `pixelsPerUnit` when the currency being used is not wei (e.g. on the livepeer_cli). We can add that in a follow-up PR if you think it's useful.